test/test_config_validation_edge_cases.py

```
@@ -0,0 +1,304 @@
+
+        schema = {
+            "type": "object",
+            "properties": {
+                "api_key": {"type": "string"}
+            },
+            "required": ["api_key"]
+        }
+
+        config = {"api_key": None}
+
+        is_valid, errors = schema_manager.validate_config_against_schema(
```

Ruff (0.14.14)

[warning] Unpacked variable `is_valid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)